packages/user/src/components/Invitation/InvitationForm.tsx

```diff
@@ -16,19 +16,27 @@ import type {
 } from "@/types";
 
 interface Properties {
+  additionalInvitationFields?: {
+    additionalDefaultValues: Record<string, any>;
+    additionalInvitationSchema: Zod.ZodObject<any>;
+    fields: React.ComponentType<{
+      useFormContext: typeof useFormContext;
+    }>;
+  };
```

Comment:
Lets define this type in src/types. It can be reused in other places too. Also have separate type for each attribute so that they can be reused in other places.